owl/EASE-SAY.owl

```diff
@@ -171,6 +404,21 @@ Image schemas are considered as fundamental, pre-conceptual, building blocks of
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-SAY.owl#InstrumentRole -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-SAY.owl#InstrumentRole">
```

Comment:
What is the plan with conflicting roles in EASE-OBJ?
As Instrument is definitely defined there.
Why not merging with it right away?

Reply:
Yes, I think we could definitely merge the corresponding roles from EASE-OBJ, e.g. Destination, Instrument, and maybe also Patient.